CONTRIBUTING.rst

```diff
@@ -172,13 +174,13 @@ This section provides general guidance for developing code for the project. The
 
 .. code:: python
 
-   determinant = A[:, :, 0:1, 0:1] * A[:, :, 1:2, 1:2] -
+   determinant = (A[:, :, 0:1, 0:1] * A[:, :, 1:2, 1:2] -
                  A[:, :, 0:1, 1:2] * A[:, :, 1:2, 0:1])
```

Comment:
maybe we can remove parenthesis to make it more clean

Reply:
syntax error then.